features/unsupported/tests/mbed/can_api/main.cpp

```diff
@@ -0,0 +1,85 @@
+#include "mbed.h"
```

Comment:
OK I will.
(by the way there are no license headers anywhere in unsupported/tests/mbed directory)